packages/asset-uploader/src/targets/s3/upload.ts

```diff
@@ -86,7 +86,8 @@ const uploadHandler = async ({
     Body: data,
     ContentType: file.contentType,
     Metadata: {
-      filename: file.filename || "unnamed",
+      // encodeURIComponent is needed to support special characters like Cyrillic
+      filename: "meta", // encodeURIComponent(file.filename) || "unnamed",
```

Comment:
I think we should either add a comment about why we preserve the old code (`encodeURIComponent`) or delete it

Reply:
Also, is it ok to loose `file.filename` here?

Reply:
Comment exists, "meta" was to test